src/main/java/com/cegeka/academy/service/challengeAnswer/ChallengeAnswerServiceImp.java

```diff
@@ -6,24 +6,46 @@
 import com.cegeka.academy.repository.UserChallengeRepository;
 import com.cegeka.academy.service.dto.ChallengeAnswerDTO;
 import com.cegeka.academy.service.mapper.ChallengeAnswerMapper;
+import com.cegeka.academy.service.userChallenge.UserChallengeService;
 import com.cegeka.academy.web.rest.errors.NotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.swing.filechooser.FileSystemView;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Date;
 
 
 @Service
 @Transactional
+@Configuration
```

Comment:
Nu este ok sa pui Configuration aici, deoarece o sa transformi acest bean in unul de tip configurare. 
Ce am vrut sa zic prin utilizarea acelor annotation era sa faci o clasa separata, sa zicem MedialFileProperties, in care sa incarci din properties acele informatii. Iar in acest serviciu sa folosesti @Autowired sa injectezi un bean de tipul clasei MediaFileProperties. Gasesti un exemplu aici: https://www.baeldung.com/configuration-properties-in-spring-boot